resources/server_install.rb

```diff
@@ -27,17 +27,20 @@
 property :password,          [String, nil], default: 'generate'
 property :port,              Integer,       default: 3306
 property :initdb_locale,     String,        default: 'UTF-8'
+property :package_name,      String,        default: node['platform_family'] == 'debian' ? "mariadb-server-#{new_resource.version}" : 'MariaDB-server'
```

Comment:
Inside the property definition there is no variable called `new_resource` only the raw property names as variables, so this should be: 
```suggestion
property :package_name,      String,        default: node['platform_family'] == 'debian' ? "mariadb-server-#{version}" : 'MariaDB-server'
```